raving-remoras/bangazon-site
# Local Delivery

# Description
Seller's can specify local delivery when selling a new product. Buyers can search by city and see what products are available for local delivery. Your own products will not be displayed.

## Related Ticket(s)
#3, #5 

## Steps to Test Solution

1. Create a new product and specify local delivery with a city and state (abbreviation). Fields should be hidden until local delivery is selected.
2. Add local delivery to another customer's product in your database so you can search.
3. Search should also work without a logged in user.

## Testing


## Documentation

## Deploy Notes
There are new columns on Product so you will need to:
1. Delete your database and migrations
2. `pyman makemigrations website`
3. `pyman migrate`
4. `pyman loaddata db.json`
